plugins/EpadDB.js

```diff
@@ -2303,7 +2302,9 @@ async function epaddb(fastify, options, done) {
     if (typeof request.body.sequence !== 'undefined') {
       sequence = request.body.sequence;
     }
-    const allStatus = ['added', 'ended', 'error', 'running'];
+
+    // const allStatus = ['added', 'ended', 'error', 'running', 'inqueue'];
+    const allStatus = ['added', 'ended', 'error'];
```

Comment:
did you remove inqueue?

Reply:
yes inqueue and running states removed to avoid to resend the plugin ids which are already running or inqueue.